packages/react-core/src/components/Nav/__tests__/NavExpandable.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { NavExpandable } from '../NavExpandable';
+import { NavItem } from '../NavItem';
+import { Nav, NavContext } from '../Nav';
+import { NavList } from '../NavList';
+
+const props = {
+  items: [
+    { to: '#link1', label: 'Link 1' },
+    { to: '#link2', label: 'Link 2' },
+    { to: '#link3', label: 'Link 3' }
+  ]
+};
+
+describe('NavExpandable', () => {
```

Comment:
Currently I think we rarely use the describe block in our revamped tests; I personally try to use it more to separate out lengthy amounts of tests that can be grouped together. For now I think we can remove the describe block and just have each test on its own.